test/lit/wat-kitchen-sink.wast

```diff
@@ -121,6 +121,9 @@
  ;; CHECK:      (type $s8 (struct (field i32) (field i64) (field $z f32) (field f64) (field (mut i32))))
  (type $s8 (struct i32 (field) i64 (field $z f32) (field f64 (mut i32))))
 
+ ;; CHECK:      (type $s9 (struct_subtype (field i32) $s2))
+ (type $s9) (sub $s2 (struct i64))
```

Comment:
```suggestion
 (type $s9 (sub $s2 (struct i64)))
```
I think?

Reply:
I would not expect wat-kitchen-sink.wast to change at all, since it tests the new parser but you modified the old parser. The new parser should already parse the standards-track format.